.travis.yml

```diff
@@ -5,8 +5,6 @@ rvm:
   - 2.1
   - 2.0
   - 1.9.3
-  - 1.9.2
-  - 1.8.7
```

Comment:
https://www.ruby-lang.org/en/news/2014/07/01/eol-for-1-8-7-and-1-9-2/

Reply:
I think we should bump the major version so we can officially drop support for these.